docs/source/Tutorial.md

```diff
@@ -892,6 +1015,39 @@ a bit more flexibility.
   auto orc = monster_builder.Finish();
 ~~~
 </div>
+<div class="language-c">
+<br>
+*Note: Since we are passing `150` as the `mana` field, which happens to be the
```

Comment:
I was copied from a specific C++ comment, so I assumed it wasn't relevant for other languages. Hence copying was the only way to also include C. Please advise.